htdocs/web_portal/components/Get_User_Principle.php

```diff
@@ -162,21 +162,29 @@ function Get_User_Principle(){
 }
 
 /**
- * Get the DN from an x509 cert or null if a user certificate can't be loaded.
- * Called from the PI to authenticate requests using certificates only.
+ * Get the DN from an x509 cert, principle from oidc auth, or null if a user certificate can't be loaded.
```

Comment:
```suggestion
 * Get the DN from an x509 cert, Principal from oidc token, or null if a user certificate can't be loaded.
```

Reply:
I missed that it's not just a user cert that can't be loaded

```suggestion
 * Get the DN from an x509 cert, Principle from oidc token, or null if neither can be loaded.
```